server/models/articles.js

```diff
@@ -42,8 +42,15 @@ const getArticleModel = (sequelize, DataTypes) => {
       allowNull: false
     },
     readTime: {
-      type: DataTypes.TIME,
+      type: DataTypes.INTEGER,
       allowNull: false
+    },
+    slug: {
+      type: DataTypes.TEXT
```

Comment:
This should be a `string` and cannot be `null`

Reply:
You said you need it to be not null, so that you can use the hooks on the article model